package.json

```diff
@@ -11,8 +11,7 @@
     "prettier": "prettier --no-semi --single-quote --write",
     "prettify": "npm run prettier -- src/*.js",
     "start": "electron .",
-    "build:macos": "electron-packager . --out=build --overwrite --platform=darwin --arch=x64 --app-version=$npm_package_version --icon=assets/icon.icns  --protocol-name=\"Email Address URL\" --protocol=\"mailto\"",
-    "build": "npm run build:macos"
+    "build": "electron-packager . --out=build --overwrite --icon=assets/icon.icns --protocol-name=\"Email Address URL\" --protocol=\"mailto\""
```

Comment:
Can you please add `--all`, so it builds all platforms? When I run this command on macOS, it only builds for `darwin`.